data/en/mbs_0111.json

```diff
@@ -1,321 +1,380 @@
 {
+    "title": "Monthly Business Survey",
+    "survey_id": "009",
     "mime_type": "application/json/ons/eq",
-    "variables": {
-        "period": "{{ format_conditional_date (answers['period-from'], exercise.start_date)}} to {{ format_conditional_date (answers['period_to'], exercise.end_date)}}"
-    },
-    "schema_version": "0.0.1",
-    "survey_id": "023",
     "theme": "default",
+    "schema_version": "0.0.1",
     "data_version": "0.0.1",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "mbs",
+    "eq_id": "mbs",
+    "form_type": "0161",
+    "view_submitted_response": {
+        "enabled": true,
+        "duration": 900
+    },
     "sections": [{
-        "id": "default-section",
-        "groups": [{
-            "blocks": [{
-                    "type": "Introduction",
-                    "id": "introduction",
-                    "primary_content": [{
+            "id": "section",
+            "groups": [{
+                "id": "group",
+                "title": "",
+                "blocks": [{
+                        "type": "Introduction",
+                        "id": "introduction",
+                        "primary_content": [{
                             "type": "Basic",
                             "id": "use-of-information",
                             "content": [{
                                 "list": [
-                                    "Please complete this statutory Question for {{respondent.trad_as_or_ru_name}}, for period {{exercise.start_date|format_date}} to {{exercise.end_date|format_date}}.",
-                                    "Data should relate to all sites in England, Scotland and Wales.",
+                                    "Data should relate to all sites in England, Scotland, Wales and Northern Ireland.",
```

Comment:
Since I'm going to request a change might as well remove Northern Ireland from here